src/OrchardCore.Modules/OrchardCore.Users/Workflows/Drivers/UserDisabledEventDisplay.cs

```diff
@@ -0,0 +1,37 @@
+using OrchardCore.DisplayManagement.Views;
+using OrchardCore.Users.Services;
+using OrchardCore.Users.Workflows.Activities;
+using OrchardCore.Users.Workflows.ViewModels;
+using OrchardCore.Workflows.Display;
+
+namespace OrchardCore.Users.Workflows.Drivers
+{
+    public class UserDisabledEventDisplay : ActivityDisplayDriver<UserDisabledEvent, UserDisabledEventViewModel>
+    {
+        public UserDisabledEventDisplay(IUserService userService)
+        {
+            UserService = userService;
+        }
+
+        protected IUserService UserService { get; }
+
+        protected override void EditActivity(UserDisabledEvent source, UserDisabledEventViewModel target)
+        {
+        }
+
+        public override IDisplayResult Display(UserDisabledEvent activity)
+        {
+            return Combine(
+                Shape($"{nameof(UserDisabledEvent)}_Fields_Thumbnail", new UserDisabledEventViewModel(activity)).Location("Thumbnail", "Content"),
```

Comment:
I think using the class name is actually bad here as you could introduce breaking changes if there was a typo fixed in the class names.

Reply:
I'm not sure to understand. Is it not using the `nameof(UserDisabledEvent)` because of that already?

Reply:
You mean that then the template name would need to be changed. But I'd expect that someone doing this understand what he's doing?

Reply:
Else that's your call we can rename this in all the other drivers too.

Reply:
Done